src/app.js

```diff
@@ -26,52 +26,55 @@ function main(sources) {
     time: new Date()
   }]);
 
+  const intent = DOM => {
+    const changeMessage$ = sources.DOM
+      .select('.input-message')
+      .events('input')
+      .map(e => e.target.value)
+      .share();
```

Comment:
would you mind explaining `share` and the need for it? I found the docs but I don't quite follow it.